projects/Bactteria_Dataset/README.md

```diff
@@ -0,0 +1,114 @@
+# Bactteria detection with darkfield microscopy Dataset
+
+This project support **`Bactteria detection with darkfield microscopy Dataset`**, and the dataset can be download from [here](https://tianchi.aliyun.com/dataset/94411).
+
+## Usage
+
+<!-- For a typical model, this section should contain the commands for training and testing. You are also suggested to dump your environment specification to env.yml by `conda env export > env.yml`. -->
+
+### Prerequisites
+
+- Python 3.8
+- PyTorch 1.10.0
+- [MIM](https://github.com/open-mmlab/mim) v0.3.4
+- [MMcv](https://github.com/open-mmlab/mmcv) v2.0.0rc3
+- [MMEngine](https://github.com/open-mmlab/mmengine) v0.1.0 or higher
+- [MMClassification](https://github.com/open-mmlab/mmclassification) v1.0.0rc4
+- [MMDetection](https://github.com/open-mmlab/mmdetection) v3.0.0rc5
+- [MMSegmentation](https://github.com/open-mmlab/mmsegmentation) v1.0.0rc3
+
+### Dataset preparing
+
+Preparing `Bactteria detection with darkfield microscopy Dataset` dataset in following format as below.
+
+```none
+  mmsegmentation
+  ├── mmseg
+  ├── tools
+  ├── configs
+  ├── data
+  │   ├── Bactteria_Det
+  │   │   ├── images
+  │   │   │   ├── train
+  |   │   │   │   ├── xxx.png
+  |   │   │   │   ├── ...
+  |   │   │   │   └── xxx.png
+  │   │   │   ├── val
+  |   │   │   │   ├── yyy.png
+  |   │   │   │   ├── ...
+  |   │   │   │   └── yyy.png
+  │   │   ├── masks
+  │   │   │   ├── train
+  |   │   │   │   ├── xxx.png
+  |   │   │   │   ├── ...
+  |   │   │   │   └── xxx.png
+  │   │   │   ├── val
+  |   │   │   │   ├── yyy.png
+  |   │   │   │   ├── ...
+  |   │   │   │   └── yyy.png
+```
+
+- download dataset from [here](https://tianchi.aliyun.com/dataset/94411) and decompression data to path 'data/Bactteria_Det'.
+- run script `"python projects/Bactteria_Dataset/tools/random_split.py"` to split dataset and change folder structure.
+
+### Training commands
+
+```shell
+mim train mmseg projects/Bactteria_Dataset/configs/${CONFIG_PATH}
+```
+
+To train on multiple GPUs, e.g. 8 GPUs, run the following command:
+
+```shell
+mim train mmseg projects/Bactteria_Dataset/configs/${CONFIG_PATH}  --launcher pytorch --gpus 8
+```
+
+### Testing commands
+
+```shell
+mim test mmseg projects/Bactteria_Dataset/configs/${CONFIG_PATH}  --checkpoint ${CHECKPOINT_PATH}
+```
+
+<!-- List the results as usually done in other model's README. [Example](https://github.com/open-mmlab/mmsegmentation/tree/dev-1.x/configs/fcn#results-and-models)
+
+You should claim whether this is based on the pre-trained weights, which are converted from the official release; or it's a reproduced result obtained from retraining the model in this project. -->
+
+## Results
+
+### Bactteria detection with darkfield microscopy Dataset
+
+|     Method      | Backbone | Crop Size |   lr   | mIoU  | mDice |                                                                        config                                                                        |
```

Comment:
It would be more convincing to provide some logs, if you don't have a server, you can put them on GitHub, something like https://github.com/Ezra-Yu/MY_STORE/releases/tag/v0.0.1